src/EnergyPlus/SingleDuct.cc

```diff
@@ -3132,10 +3185,36 @@ void SingleDuctAirTerminal::SizeSys(EnergyPlusData &state)
                 MinAirFlowFracDes = 0.0;
             }
         }
+        if (SizingDesRunThisAirSys) {
+            if (state.dataSize->SysSizInput(SysSizNum).SystemOAMethod == SOAM_SP) { // 62.1 simplified procedure
+                if (this->MaxAirVolFlowRate > 0.0) {
+                    FixedMinAirDes = 1.5 * max(state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).VozClgByZone,
+                                               state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).VozHtgByZone);
+
+                    // adjust maximum flow rate
+                    if (FixedMinAirDes > this->MaxAirVolFlowRate && IsMaxFlowAutoSize) {
+                        this->MaxAirVolFlowRate = FixedMinAirDes;
+                    } else if (FixedMinAirDes > this->MaxAirVolFlowRate) {
+                        ShowWarningError(state, "SingleDuctSystem:SizeSys: Maximum air flow rate for " + this->SysName + " is potentially too low.");
+                        ShowContinueError(
+                            state,
+                            "The flow is lower than the minimum flow rate calculated following the ASHRAE Standard 62.1 Simplified Procedure:");
+                        ShowContinueError(state, format(" User-specified maximum air flow rate: {:.3R} m3/s.", this->MaxAirVolFlowRate));
+                        ShowContinueError(state, format(" Calculated minimum air flow rate: {:.3R} m3/s.", FixedMinAirDes));
+                        FixedMinAirDes = this->MaxAirVolFlowRate;
+                    }
```

Comment:
Handles cases where the SP flow is greater than the auto/hard-size maximum. Increase the autosized maximum but cap to the hard-sized maximum and issue a warning.